### DISCUSSION TITLE
Slack

### DISCUSSION THREAD

**Discussion by igoralves1** [EHealthBR/ehealth-book#22] [Ideas]
Olá,
existe algum canal de slack para esse projeto?

**Comment by maltempi** [comment]
Oi @igoralves1! 

Ainda não temos um canal no Slack. Como temos muitos profissionais envolvidos que não são necessariamente de TI, o canal mais viável para nós hoje é no WhatsApp. Já estamos em uns 15.

Você pode entrar lá através desse link: https://chat.whatsapp.com/IveedBybMzm3mbpR3Gfbc3

Obrigado pelo contato e te espero lá no nosso grupo no WhatsApp.
